lib/sqlalchemy/sql/sqltypes.py

```diff
@@ -934,6 +934,9 @@ def process(value):
     # both sqlite3 and pg8000 seem to return it,
     # psycopg2 as of 2.5 returns 'memoryview'
     def result_processor(self, dialect, coltype):
+        if dialect.supports_native_bytes:
```

Comment:
**mike bayer** (zzzeek) wrote:

lets call it "returns_native_bytes" since we dont seem to be affecting the bind side

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4577

Reply:
4f2fd88bd9af54c54438a3b72a2f30384b0f8898